src/region_growing/region_growing.py

```diff
@@ -20,20 +23,33 @@ def __init__(self, threshold_angle=20, threshold_curve=1.0, max_nn=30,
         self.grow_region_knn = grow_region_knn
         self.grow_region_radius = grow_region_radius
 
-    def set_input_cloud(self, las, seed_point_label, mask):
-        """ Function to convert to o3d point cloud. """
-        coords = np.vstack((las.x[mask], las.y[mask], las.z[mask])).transpose()
-        pcd = o3d.geometry.PointCloud()
-        pcd.points = o3d.utility.Vector3dVector(coords)
-        self.pcd = pcd
+        self.exclude_labels = exclude_labels
+        self.seed_point_label = label  # TODO misschien is het mooier om .get_label() te gebruiken.
```

Comment:
Gewoon initialiseren als locale global var? Beetje onnodige todo denk ik nu

Reply:
Nu het een subclass is van die abstract class, kun je ook overal ipv `self.seed_point_label` gewoon `self.label` gebruiken toch?